### PR TITLE
feat(components): [message-box] add loading-icon to confirmbtn,cancelbtn

### LINKED ISSUE #17358: [Component] [message-box] MessageBox 自定义个性化消息弹框 beforeClose 回调 instance 建议添加确认按钮自定义 Loading 图标配置
Bug Type: **`Component`**

## Environment

- Vue Version: `3.4.21`
- Element Plus Version: `2.7.6`
- Browser / OS: `Edge 126.0.2592.68`
- Build Tool: `Vite`

### Related Component

- `el-message-box`

### Reproduction Link

[Link]()

### Steps to reproduce

MessageBox 自定义个性化消息弹框 beforeClose 回调 instance 建议添加确认按钮自定义 Loading 图标配置

### What is Expected?

image

### What is actually happening?

/

### Additional comments

_(empty)_

### PR DESCRIPTION
**Pull request by wzc520pyfm**
closed #17358

image

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by btea — APPROVED** [review]
LGTM!
The documentation should probably be updated as well.

File: docs/en-US/component/message-box.md [hunk 1]
```diff
@@ -177,6 +177,8 @@ The corresponding methods are: `ElMessageBox`, `ElMessageBox.alert`, `ElMessageB
...
 | confirm-button-text          | text content of confirm button                                                                                                           | ^[string]                                                                          | OK                                               |
+| cancel-button-loading-icon   | loading icon content of cancel button                                                                                                    | ^[string] / ^[Component]                                                           | Loading                                          |
+| confirm-button-loading-icon  | loading icon content of confirm button                                                                                                   | ^[string] / ^[Component]                                                           | Loading                                          |
```
Comment (btea) [inline docs/en-US/component/message-box.md:181]: Please add a version number.

Comment (wzc520pyfm) [reply docs/en-US/component/message-box.md]: I have added the version for new options, but ci fails for some unknown reason.🤔

**Comment by btea** [comment]
https://github.com/pnpm/action-setup/issues/135#issuecomment-2206861174

File: docs/en-US/component/message-box.md (hunk 1 above)
Comment (btea) [reply docs/en-US/component/message-box.md]: https://github.com/pnpm/action-setup/issues/135#issuecomment-2206861174

**Review by tolking — APPROVED** [review]

**Pull request merged**